src/lang/english.txt

```diff
@@ -2772,6 +2772,7 @@ STR_SAVELOAD_DETAIL_CAPTION                                     :{BLACK}Game Det
 STR_SAVELOAD_DETAIL_NOT_AVAILABLE                               :{BLACK}No information available
 STR_SAVELOAD_DETAIL_COMPANY_INDEX                               :{SILVER}{COMMA}: {WHITE}{STRING1}
 STR_SAVELOAD_DETAIL_GRFSTATUS                                   :{SILVER}NewGRF: {WHITE}{STRING}
+STR_SAVELOAD_DIFFERENT_ID										:{RED}This savegame is not the same as the one being saved.
```

Comment:
Bad indentation of the string itself.
Additionally, this string reads weird to me. I think it would be better to use just "game" rather than "savegame". Suggestion: "Not the same game as you are saving."